README.md

```diff
@@ -85,6 +85,57 @@ defmodule MyAppWeb.UserController do
 end
 ```
 
+### Associations
+
+Based on the following schemas:
+
+```elixir
+defmodule MyApp.Post do
+  use Ecto.Schema
+
+  schema "posts" do
+    field(:title, :string)
+
+    belongs_to(:author, MyApp.User)
+    belongs_to(:category, MyApp.Category)
+  end
+end
+
+defmodule MyApp.User do
+  use Ecto.Schema
+
+  schema "users" do
+    field(:name, :string)
+    field(:email, :string)
+
+    has_many(:posts, MyApp.Post, foreign_key: :author_id)
+  end
+end
+
+defmodule Seeker.Schemas.Category do
+  use Ecto.Schema
+
+  schema "categories" do
+    field(:name, :string)
+
+    has_many(:posts, MyApp.Post)
+  end
+end
+```
+
+Posts can be filtered and sorted using double underscore (`__`) in
+the params like this:
```

Comment:
this is a weird syntax, where did it comes from?

Reply:
Consider in the future a more standard query similar at what JSON API or Graphql does